Models/NHES/Systems/Examples/NuScale_Coupling_4_loop_control_Pressurizer_CR.bak-mo

```diff
@@ -0,0 +1,91 @@
+within NHES.Systems.Examples;
```

Comment:
Unpaired .bak-mo file. Do we need to change its extension?

Reply:
Taken out.